CHANGELOG.md

```diff
@@ -8,24 +8,42 @@ All notable changes to this project will be documented in this file. This projec
 
 ### Incompatible Changes
 
--   Updated theme to reflect the latest adapations made in our applications. The old behavior can be restored by adding the following to the theme:
+-   Removed deprecated package `@comet/admin-date-picker`
+-   The minimum version of `react` and `react-dom` has been changed to 17.0
+-   Migrated from "Material-UI" v4 to "MUI" v5
+    -   The default color of `MuiButton` and `MuiIconButton` was [changed](https://mui.com/guides/migration-v4/#button) from "default" to "primary", to restore the previous "default" style, set the color to "info"
+    -   See the official [MUI Migration Guide](https://mui.com/guides/migration-v4/) for more details
+-   Changed the styles-engine from styled-components to emotion
```

Comment:
```suggestion
-   Changed the styled engine from styled-components to emotion
```

Maybe we could add a note on how to change it back to styled-components? https://mui.com/guides/interoperability/#change-the-default-styled-engine

Reply:
I don't think there will ever be a need to change it back. From my experience it's pretty much the same...